### PR TITLE
Add UI

### PR DESCRIPTION
**Pull request by abeltsew**
Dear Reviewer,

In this PR the following UI parts have been implemented.

- [x] List all books.
- [x] List all people.
- [x] Create a person (teacher or student, not a plain Person).
- [x] Create a book.
- [x] Create a rental.
- [x] List all rentals for a given person id.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    print "teacher's specialization: "
+    specialization = gets.chomp
+    print "teacher's age: "
+    age = gets.chomp
+    print "teacher's name: "
+    name = gets.chomp
+    teacher = Teacher.new(specialization, age, name, parent_permission: true)
+    @people << teacher
+    puts 'You have successfully registered a Teacher'
+  end
```
Comment (KanzaTahreem) [inline app.rb]: - [x] Kindly note that the teacher's attributes are misplaced. In place on `name`, teacher's `age` is displaying, and in place of `age`, I can see `specialization` ⚠Also, teacher's `name` is not present. Kindly fix these issues for getting the expected behaviour of app as given in the project requirements 🤓⛏

  image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @abeltsew 

Good job so far! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good UI
- ✅ Descriptive PR 
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

Cheers and Happy coding!👏👨‍💻💻

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (abeltsew) [reply app.rb]: Thank you,

It was a bug created by the order of parameters.

It has been addressed.

Thank you

**Review by binhussen — APPROVED** [review]
Hi @abeltsew , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Added Console UI ✔️
- Well STractured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**